sql/2020/04_Fonts/04_02.web_fonts_usage_by_country.sql

```diff
@@ -0,0 +1,27 @@
+#standardSQL
+#web_fonts_usage_by_country
+SELECT
+ _TABLE_SUFFIX AS client,
+ country,
+ COUNT(0) AS freq_url,
+ ROUND(APPROX_QUANTILES(bytesFont, 1000)[OFFSET(500)] / 1024, 2) AS median_font_bytes
+FROM (
+SELECT
+ DISTINCT origin,
+ `chrome-ux-report`.experimental.GET_COUNTRY(country_code) AS country
+FROM 
+  `chrome-ux-report.materialized.country_summary`
+WHERE 
+  yyyymm = 202008)
+JOIN 
+  `httparchive.summary_pages.2020_08_01_*`
+ON 
+  CONCAT(origin, '/') = url
+WHERE 
+  bytesFont IS NOT NULL
+GROUP BY
+ client,
+ country
+ORDER BY
+ client,
+ country
```

Comment:
When joining data from the Chrome UX Report and HTTP Archive datasets, also be sure to join by the device/client.

This suggestion includes that change plus some minor whitespace fixes and the`ROUND`/kbytes feedback from the previous query.

```suggestion
#standardSQL
#web_fonts_usage_by_country
SELECT
  _TABLE_SUFFIX AS client,
  country,
  COUNT(0) AS freq_url,
  APPROX_QUANTILES(bytesFont, 1000)[OFFSET(500)] / 1024 AS median_font_kbytes
FROM (
  SELECT
    DISTINCT origin,
    device,
    `chrome-ux-report`.experimental.GET_COUNTRY(country_code) AS country
  FROM 
    `chrome-ux-report.materialized.country_summary`
  WHERE 
    yyyymm = 202008)
JOIN 
  `httparchive.summary_pages.2020_08_01_*`
ON 
  CONCAT(origin, '/') = url AND
  IF(device = 'desktop', 'desktop', 'mobile') = _TABLE_SUFFIX
WHERE 
  bytesFont IS NOT NULL
GROUP BY
  client,
  country
ORDER BY
  client,
  country
```

Reply:
Thank you to point out the procedure of combining CrUX report and HTTP Archive dateset.